### ISSUE TITLE
Users Manager : Functionnality not found

### ISSUE THREAD

**Issue by Max-ADC** [linagora/linshare#93]
Hello,

I have a problem when accessing to the manage user screen of a guest user (Users > Manages users). I Have the followingg error : "The functionality was not found. It may not be present in the selected domain."
I can access to that screen for a domain users with no problems.
Does anyone know what could be the problem ?

My versions : core 2.3.2 / ui-admin 3.3.1 / ui-user 2.3.2

Regards,

**Comment by fmartin-linagora** [comment]
Hello,
Yes, I have the same issue, I'm looking into to find a solution ASAP :( 
Fred.

**Comment by fmartin-linagora** [comment]
Hi,
I found the issue, when a guest is created, the user interface  use the parameters/functionalities from the domain of the account (aka owner) that is creating the guest. It allows us to have different configurations, like a regular user domain and a vip user domain with distinct expiration policy.
The admin interface is loading the configuration from the guest's domain instead of the owner's domain. It seams to be an old bug, it just appears now because I recently fixed the backend. :(
Fred.
